pages/Getting Started/Installation.md

```diff
@@ -70,6 +70,12 @@ or a tagged release from the arch packages:
 sudo pacman -S hyprland
 ```
 
+Alternatively, install the `hyprland-meta` package to get the entire Hypr ecosystem.
```

Comment:
needs a mention that it will _compile_ and pull the latest git, which may not be ultra stable

Reply:
Done